timemachine/md/builders.py

```diff
@@ -50,16 +55,16 @@ def build_protein_system(host_pdbfile):
     return solvated_host_system, solvated_host_coords, nwa, nha, box, modeller.topology
 
 
-def build_water_system(box_width):
-    ff = app.ForceField("tip3p.xml")
+def build_water_system(box_width, tm_ff: Forcefield):
+    ff = app.ForceField(f"{tm_ff.water_ff}.xml")
```

Comment:
nit: seems like it might be more precise to pass in just `water_ff` and `sanitized_water_ff` here instead of the full `Forcefield` object

Reply:
updated